### DISCUSSION TITLE
Version 3.2.97 breaks copying form fields when viewing them

### DISCUSSION THREAD

**Discussion by lucacastelnuovo** [filamentphp/filament#14175] [Help]
### Package

Panel builder

### Package Version

v3.2.97

### How can we help you?

Please see the video.

Between version `3.2.96` and `3.2.97` something changed causing the `Markdowneditor` field to not be copiable when viewing the resource in a relation manager.

https://github.com/filamentphp/filament/compare/v3.2.96...v3.2.97

https://github.com/user-attachments/assets/9d8c91a6-3216-4925-b905-31bb19e23c5d

**Comment by lucacastelnuovo** [comment]
The same problem occurs with `RichEditor`, `TextArea` and `TextInput`. `Select` fields seem not to be affected.

**Reply by leandrocfe** [reply]
Did you update Filament version?
Same issue?

**Reply by lucacastelnuovo** [reply]
Yes I did, currently running v3.2.111 and the issue still persists.
